iocage/lib/Jail.py

```diff
@@ -0,0 +1,409 @@
+import iocage.lib.JailConfig
+import iocage.lib.Network
+import iocage.lib.Storage
+import iocage.lib.Releases
+import iocage.lib.Release
+import iocage.lib.helpers
+
+import iocage.lib.ZFSBasejailStorage
+import iocage.lib.NullFSBasejailStorage
+import iocage.lib.StandaloneJailStorage
+
+import libzfs
+import subprocess
+import uuid
+
+class Jail:
+
+  def __init__(self, data = {}, zfs=None, host=None, logger=None):
```

Comment:
that would be helpful

both of those.

it makes reading code a lot easier if emacs isn't screaming at me even before i've done anything :D